elastic/kibana
# [RAM] Fix bulk action for o11y for 7.17

## Summary

In branch 7.17, we discover through a [SDH](https://github.com/elastic/sdh-kibana/issues/2584) that we had a bug around bulk actions for o11y. This PR will allow to fix it.

I will un-skip the test only for v7.17.2 since o11y is not using anymore the bulk actions functionality in the 8 series.

To test this PR in main:

1. Change this attribute `showCheckboxes` to true in this [file](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability/public/pages/alerts/containers/alerts_table_t_grid/alerts_table_t_grid.tsx#L397) 
2.  Unskip the test `Bulk Actions` here `x-pack/test/observability_functional/apps/observability/alerts/index.ts`
3.  Run `node ./scripts/functional_tests_server.js --config x-pack/test/observability_functional/with_rac_write.config.ts`
4.  Run `node scripts/functional_test_runner --config x-pack/test/observability_functional/with_rac_write.config.ts`

And you should see that everything is working as expected

### Checklist
